lib/asset_manager_attachment_metadata_updater.rb

```diff
@@ -0,0 +1,16 @@
+class AssetManagerAttachmentMetadataUpdater
+  def self.update_range(start, finish)
+    AttachmentData.find_each(start: start, finish: finish) do |attachment_data|
```

Comment:
I have a vague recollection that there are some `AttachmentData`'s (in integration at least) which have no underlying file on the file system. We should ensure this will run without errors in such a scenario.